plugins/generator-1.19.4/forge-1.19.4/templates/json/advancement.json.ftl

```diff
@@ -0,0 +1,65 @@
+<#-- @formatter:off -->
+<#include "../mcitems.ftl">
+{
+    <#if !data.disableDisplay>
+        "display": {
+          <#if data.parent == "none" && !data.parent.toString().contains("@")>
+              <#if !data.background?has_content || data.background == "Default">
+                  "background": "minecraft:textures/block/stone.png",
+              <#else>
+                  "background": "${modid}:textures/screens/${data.background}",
+              </#if>
+          </#if>
+          "icon": {
+            ${mappedMCItemToIngameItemName(data.achievementIcon)}
+          },
+          "title": {
+            "translate": "advancements.${registryname}.title"
+          },
+          "description": {
+            "translate": "advancements.${registryname}.descr"
+          },
+          "frame": "${data.achievementType}",
+          "show_toast": ${data.showPopup},
+          "announce_to_chat": ${data.announceToChat},
+          "hidden": ${data.hideIfNotCompleted}
+        },
+    </#if>
+    "criteria": {
+      "${registryname}": ${triggercode}
+    },
+    "requirements": [

```

Comment:
Why is this section only present for FG and not for DP generator?

Anyway it should not be present, as there is no need to specify itself as a requirement

Reply:
✔️ Fixed